src/zenml/cli/workspace.py

```diff
@@ -189,3 +189,21 @@ def delete_workspace(workspace_name_or_id: str) -> None:
     except (KeyError, IllegalOperationError) as err:
         cli_utils.error(str(err))
     cli_utils.declare(f"Deleted workspace '{workspace_name_or_id}'.")
+
+
+@cli.group(cls=TagGroup, tag=CliCategories.MANAGEMENT_TOOLS)
+def project() -> None:
+    """Deprecated commands for project management."""
+    cli_utils.warning(
+        "DEPRECATION WARNING: `project` has been renamed to"
+        "`workspace`. Please switch to using "
+        "`zenml workspace ...`."
+    )
+
+
+project.add_command(list_workspaces)
```

Comment:
Also here not sure whether you can add commands from other groups like this. Awesome if it works but please check manually to make sure both versions work with this.

Reply:
Found it here: https://stackoverflow.com/a/45514541 - also works in my tests

Reply:
Awesome